liblz4_vendor/CHANGELOG.rst

```diff
@@ -5,6 +5,11 @@ Changelog for package liblz4_vendor
 0.26.5 (2024-09-06)
 -------------------
 
+0.26.6 (2024-12-18)
+-------------------
+* [Jazzy] Release 0.26.5 (`#1800 <https://github.com/ros2/rosbag2/issues/1800>`_)
+* Contributors: Marco A. Gutierrez
+
```

Comment:
So I think that the previous tag wasn't done correctly, which ends up confusing `catkin_generate_changelog`.  In this case, we don't need to mark the fact that we released 0.26.5 (that is obvious), and this section should be *higher* in the file than 0.26.5.  My suggestion is to make this an empty section and move it above the 0.26.5 section above.

The same goes for all of the other changelog sections below.

Reply:
Oh I didn't realize, thanks! I'll reorder the changelog info and remove the unnecessary stuff there.

It seems that the tag is not in the jazzy branch and that caused `catking_generat_changelog` to misbehave. I guess we can live with it since after this release we'll have a new tag. I'll make sure the tag lands on the `jazzy` branch.